opencog/guile/LoggerSCM.cc

```diff
@@ -140,29 +170,42 @@ LoggerSCM::LoggerSCM() : ModuleWrap("opencog logger") {}
 /// Thus, all the definitions below happen in that module.
 void LoggerSCM::init(void)
 {
-	define_scheme_primitive("cog-logger-set-level!",
+	define_scheme_primitive("cog-default-logger",
+		&LoggerSCM::do_default_logger, this, "logger");
+	define_scheme_primitive("cog-ure-logger",
+		&LoggerSCM::do_ure_logger, this, "logger");
```

Comment:
Regarding doing
```scheme
(define my-fav-logger (create-new-logger "/path/to/log/file"))
(cog-logger-set-loglevel my-fav-logger  "fine")
```
yes, that's the idea, it's just I didn't implement `create-new-logger` yet, and also it doesn't change the fact that I need to be able to reference the existing loggers created on the C++ side. So the plan is to have both.